website/docs/d/alert_configuration.html.markdown

```diff
@@ -91,12 +91,14 @@ data "mongodbatlas_alert_configuration" "test" {
 
 * `project_id` - (Required) The ID of the project where the alert configuration will create.
 * `alert_configuration_id` - (Required) Unique identifier for the alert configuration.
+* `output` - (Optional) List of formatted output requested for this alert configuration
```

Comment:
can you also add an example on this file so users know how to use new data source?

Reply:
I added an example in `examples/atlas-alert-configurations` which contains a README.md describing how to use, it details how to
- Basic usage of the data_source
- Using the data_source to add all alerts as hcl and importing them into state file